design/ceph/object-store.md

```diff
@@ -62,6 +59,8 @@ The object store settings are exposed to Rook as a Custom Resource Definition (C
 
 The pools are the backing data store for the object store and are created with specific names to be private to an object store. Pools can be configured with all of the settings that can be specified in the [Pool CRD](/Documentation/ceph-pool-crd.md). The underlying schema for pools defined by a pool CRD is the same as the schema under the `metadataPool` and `dataPool` elements of the object store CRD. All metadata pools are created with the same settings, while the data pool can be created with independent settings. The metadata pools must use replication, while the data pool can use replication or erasure coding.
```

Comment:
not sure how the private pools works. since the pools are at zone level, what happens if multiple objects stores with pools are added (with the same default zone). will they share the pools or will different pools be created on the same zone?

Reply:
The CephObjectStore for backward compatibility can still be used to define the simple case to create a local object store that won't be multi-site enabled. The new zone CRD should represent the pools that will be created and which could be shared by multiple object stores.

Reply:
@yuvalif , the pool will be shared in that case. The pull are created automatically when RGW starts and it cannot find it, so the first RGW will create them (per zone).

Reply:
@travisn basically if the user didn't provide multisite config, RGW uses "default" for the zone name (also for the zone group name).
So the pools that will be created will be default.rgw.control, ...
This means that when you add multi-site it will be in effect only on new buckets and objects. @alimaredia can you add a short paragraph about it. There is a procedure to migrating existing data by copying the pools (as we cannot rename them).

Reply:
I see we create a realm/zonegroup/zone today for each rgw, so that behavior will stay as long as the user didn't supply any multisite configuration.